packages/server/utils/stripe/StripeManager.ts

```diff
@@ -72,28 +72,20 @@ export default class StripeManager {
     return this.stripe.subscriptions.del(stripeSubscriptionId)
   }
 
-  async updateSubscriptionQuantity(
-    stripeSubscriptionId: string,
-    quantity: number,
-    prorationDate?: number
-  ) {
-    return this.stripe.subscriptions.update(stripeSubscriptionId, {
-      quantity,
-      proration_date: prorationDate
+  async getSubscriptionItem(subscriptionId: string) {
+    const allSubscriptionItems = await this.stripe.subscriptionItems.list({
```

Comment:
Stripe now has a [SubscriptionItem](https://stripe.com/docs/api/subscription_items) object and `quantity` no longer exists on the [subscription](https://stripe.com/docs/api/subscriptions/object) object. We only have one `subscriptionItem` per `subscription`, so when we want to get the subscription quantity, we can call `getSubscriptionItem`. 

I named it `get` instead of `retreiveSubscriptionItem` to signal that there's some additional logic going on here, unlike the other API calls that simply reach out to Stripe.